config/samples/misf_v1alpha1_managedfusionoffering.yaml

```
@@ -6,6 +6,6 @@ metadata:
 spec:
   kind: DF
   release: "4.12"
-  config: 
+  config: |
     usableCapacityInTiB: "4"
```

```suggestion
    usableCapacityInTiB: 4
```
usableCapacityInTiB's value should be an int and not a string